### PR TITLE
Features in Readme

### PR DESCRIPTION
**Pull request by normanrz**
### Mailable description of changes:
 - Updated Readme with Feature list

------
- [x] Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,29 +123,44 @@ See: http://www.scala-sbt.org/release/docs/Getting-Started/Setup.html
...
 ### Run on a remote machine
 ```bash
 sbt "run -Dhttp.uri=http://<remote address>:9000"
 ```
-Will fetch all Scala, Java and node dependencies and run the application on Port 9000. Make sure that the mongoDB service is running before you start sbt.
+Will fetch all Scala, Java and node dependencies and run the application on Port 9000. 
+Make sure that the PostgreSQL service is running before you start sbt.
 
-Make sure to open port `9000` in our firewall. This is only recommended for development purposes. See below for a recommended production setup.
+Make sure to open port `9000` in our firewall.
```
Comment (jstriebel) [inline README.md]: … Your firewall

File: README.md [hunk 2]
```diff
@@ -1,11 +1,45 @@
...
+* Fully browser-based user experience with efficient data streaming
+* Creation/editing of skeleton and volume annotations
+* [Innovative flight mode for fast skeleton tracing](https://www.nature.com/articles/nmeth.4331)
+* Optimized performance for large tracings
+* User and task management for high-throughput crowdsourcing
+* Sharing and collaboration features
+* [Standalone datastore component](https://github.com/scalableminds/webknossos/tree/master/webknossos-datastore) for flexible deployments
+* [Supported dataset formats: WKW (Optimized), KNOSSOS cubes](https://github.com/scalableminds/webknossos/wiki/Datasets)
+* Supported image formats: Grayscale, Segmentation Maps, RGB, Multi-Channel
+* [Documented frontend API for user scripts](https://demo.webknossos.org/assets/docs/frontend-api/index.html), REST API for backend access
```
Comment (jstriebel) [inline README.md]: All functions seem to be documented twice here? Not important, though.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,45 @@
 # webKnossos
-Cellular-resolution connectomics is currently substantially limited by the throughput and efficiency of data analysis. Current solutions require an efficient integration of automated image analysis with massive manual data annotation. To scale such annotation efforts it is decisive to be able to crowd source data analysis online. Here we present **webKnossos** (former oxalis).
+Cellular-resolution connectomics is currently substantially limited by the throughput and efficiency of data analysis. 
+Current solutions require an efficient integration of automated image analysis with massive manual data annotation. 
+To scale such annotation efforts it is decisive to be able to crowd source data analysis online. 
+Here we present **webKnossos**.
+
+> Boergens Berning Bocklisch Bräunlein Drawitsch Frohnhofen Herold Otto Rzepka Werkmeister Werner Wiese Wissler & Helmstaedter
```
Comment (jstriebel) [inline README.md]: Does not look nice. Maybe remove add commas or remove authors?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,45 @@
...
+
+## Features
+* Exploration of large 3D image datasets
+* Fully browser-based user experience with efficient data streaming
+* Creation/editing of skeleton and volume annotations
+* [Innovative flight mode for fast skeleton tracing](https://www.nature.com/articles/nmeth.4331)
+* Optimized performance for large tracings
+* User and task management for high-throughput crowdsourcing
+* Sharing and collaboration features
+* [Standalone datastore component](https://github.com/scalableminds/webknossos/tree/master/webknossos-datastore) for flexible deployments
```
Comment (jstriebel) [inline README.md]: maybe use relative links?

File: README.md (hunk 2 above)
Comment (normanrz) [reply README.md]: Yeah, that's because there are two API versions (`2` and `latest`)

File: README.md (hunk 4 above)
Comment (normanrz) [reply README.md]: I think that doesn't really work, if you open https://github.com/scalableminds/webknossos/blob/readme/README.md

**Pull request merged**